### PR TITLE
Mta rebranding

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,18 @@
 2. Follow the following steps (Minimum Python-3.6 required)
 
 ```shell
-git clone https://github.com/<user>/rhamt_integration_test.git
-cd rhamt_integration_test
-python3 -m venv .rhamt_venv
-source .rhamt_venv/bin/activate
+git clone https://github.com/<user>/mta_integration_tests.git
```
Comment (digitronik) [inline README.md:9]: Required: `windup_integration_test` ^^

File: mta/entities/__init__.py [hunk 2]
```diff
@@ -55,20 +57,55 @@ def is_empty(self):
...
+
+    def clear_search(self):
+        """Clear search"""
+        if self.search.value:
+            self.search.fill("")
+
+    @property
+    def is_displayed(self):
+        return self.is_empty or (
+            self.new_project_button.is_displayed and self.title.text == "Projects"
```
Comment (digitronik) [inline mta/entities/__init__.py:88]: Do you want this changes in rebranding PR?

File: mta/test_framework/pytest_plugin.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import logging
+
+logger = logging.getLogger("mta.test_framework.pytest_plugin")
```
Comment (digitronik) [inline mta/test_framework/pytest_plugin.py:3]: We are not using logger in this this file.

Comment (digitronik) [reply mta/test_framework/pytest_plugin.py:3]: ahhh It was old stuff... Understand instate of renaming you deleted this file thats why github showing it. :+1:

File: mta/entities/__init__.py (hunk 2 above)
Comment (sshveta) [reply mta/entities/__init__.py:88]: I was working on this and then rebranding came up .. I will keep this for separate PR .

**Pull request merged**